packages/jest-message-util/src/__tests__/messages.test.ts

```diff
@@ -27,63 +28,76 @@ const unixStackTrace =
   at Spec.addResult (../jest-jasmine2/build/jasmine-2.4.1.js:333:34)
   at Expectation.addResult (../jest-jasmine2/build/jasmine-2.4.1.js:591:21)
   at Expectation.toBe (../jest-jasmine2/build/jasmine-2.4.1.js:1504:12)
-  at Object.it (build/__tests__/messages-test.js:45:41)
+  at Object.it (/build/__tests__/messages-test.js:45:41)
   at Object.<anonymous> (../jest-jasmine2/build/jasmine-pit.js:35:32)
   at attemptAsync (../jest-jasmine2/build/jasmine-2.4.1.js:1919:24)`;
 
-const assertionStack =
-  '  ' +
-  `
-    Expected value to be of type:
-      "number"
-    Received:
-      ""
-    type:
-      "string"
-
-      at Object.it (__tests__/test.js:8:14)
-      at Object.asyncFn (node_modules/jest-jasmine2/build/jasmine_async.js:124:345)
-      at resolve (node_modules/jest-jasmine2/build/queue_runner.js:46:12)
+const assertionStack = stripIndent`
+  Expected value to be of type:
+    "number"
+  Received:
+    ""
+  type:
+    "string"
```

Comment:
Jest's own test suite convinced me that it was necessary to preserve all whitespace in error messages, which means that good looking results require having input that is formatted in a reasonable way.